recipes/datajoint/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "datajoint" %}
+{% set version = "0.11.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a35e8d87398b813aef95f4a1bfbaacae5cc905971d6cb99d3114ab2c0f35229b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python>=3.4
```

There needs to be a space here else feedstock conversion will fail
```suggestion
    - python >=3.4
```
(there should not be a space between `>=` and `3.4`)